iree/compiler/Dialect/HAL/Conversion/HALToVM/ConvertCommandBufferOps.cpp

```diff
@@ -40,6 +40,7 @@ class CommandBufferFillBufferOpConversion
     // Record the original pattern length then extend it to a 32 bit integer.
     auto originalPatternType = op.pattern().getType();
     auto patternBitWidth = originalPatternType.getIntOrFloatBitWidth();
+    if (patternBitWidth < 8) patternBitWidth = 8;
```

Comment:
Might need some comment around this. It's fragile to me because we're adding special logics to i1 cases in many places.

Reply:
Ok, added some comments.